Mathlib/Topology/Algebra/Module/Alternating.lean

```diff
@@ -0,0 +1,616 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov, Heather Macbeth, Sébastien Gouëzel
+-/
+import Mathlib.LinearAlgebra.Alternating
+import Mathlib.Topology.Algebra.Module.Multilinear
+
+/-!
+# Continuous alternating multilinear maps
+
+In this file we define bundled continuous alternating maps and develop basic API about these
+maps, by reusing API about continuous multilinear maps and alternating maps.
+
+## Keywords
+
+multilinear map, alternating map, continuous
+-/
+
+open Function Matrix
+
+open scoped BigOperators
+
+/-- A continuous alternating map is a continuous map from `ι → M` to `N` that is
+
+- multilinear : `f (update m i (c • x)) = c • f (update m i x)` and
+  `f (update m i (x + y)) = f (update m i x) + f (update m i y)`;
+- alternating `f v = 0` whenever `v` has two equal coordinates.
+-/
+structure ContinuousAlternatingMap (R M N ι : Type _) [Semiring R] [AddCommMonoid M] [Module R M]
+    [TopologicalSpace M] [AddCommMonoid N] [Module R N] [TopologicalSpace N] extends
+    ContinuousMultilinearMap R (fun _ : ι => M) N where
+  map_eq_zero_of_eq' : ∀ (v : ι → M) (i j : ι), v i = v j → i ≠ j → toFun v = 0
+
+notation "Λ^" ι "⟮" R "; " M "; " N "⟯" => ContinuousAlternatingMap R M N ι
```

Comment:
I have a problem with this notation: usually, `Λ^n(E)` is the alternating space of `E`, based on `n`-tensors of elements of `E` quotiented by the relation that kills pure tensors with two identical components. This is not a space of functions on `E^n` -- the space you are using is based on the dual space of `E`, and would usually by denoted by `Λ^n(E^*)`. In the same way that we have `E [×n]→L[𝕜] F` for continuous multilinear maps, maybe use `E [Λ, ×n]→L[𝕜] F` to lift the ambiguity? This would probably be worth discussing on Zulip, because other people may have strong opinions on this. Except that this notation only works well for `fin n`, not so much for a general indexing type...

Reply:
I agree that `Λ^n(E)` is not the same as `Λ^n(E^*)` so this isn't yet the right notation.

Reply:
I changed notation to `E [Λ^n]→L[𝕜] F` as suggested by @sgouezel on Zulip. Should I change it to `E [Λ, ×n]→L[𝕜] F`?